src/uct/ib/rdmacm/rdmacm_ep.h

```diff
@@ -11,10 +11,12 @@
 struct uct_rdmacm_ep {
     uct_base_ep_t                      super;
     void                               *priv_data;
-    ucs_list_link_t                    list_elem;
+    ucs_list_link_t                    list_elem;       /* for the pending_eps_list*/
     struct sockaddr_storage            remote_addr;
     int                                is_on_pending;
+    int                                is_on_cm_ids_list;
     uct_worker_cb_id_t                 slow_prog_id;
+    uct_rdmacm_cm_id_t                 *rdmacm_cm_id;
```

Comment:
rename to 'cm_id'

Reply:
but there is also a cm_id inside it

Reply:
can it be some other short name?

Reply:
rdmacm_ctx?

Reply:
cm_id_ctx?
(and change the struct/type name accordingly)